packages/fast-tooling-react/src/css-editor/editor.spec.tsx

```diff
@@ -1,141 +1,39 @@
-        const topValue: string = "1";
-        const leftValue: string = "2";
-        const bottomValue: string = "3";
-        const rightValue: string = "4";
-        const callback: any = jest.fn();
-        const data: CSSEditorValues = {
-            marginTop: topValue,
-            marginLeft: leftValue,
-            marginBottom: bottomValue,
-            marginRight: rightValue,
-            paddingTop: topValue,
-            paddingLeft: leftValue,
-            paddingBottom: bottomValue,
-            paddingRight: rightValue,
-        };
-
-        const rendered: any = mount(<CSSEditor data={data} onChange={callback} />);
-
-        const cssPosition: any = rendered.find("CSSSpacing");
-
-        expect(cssPosition.prop("data")[SpacingProperty.marginTop]).toBe(topValue);
-        expect(cssPosition.prop("data")[SpacingProperty.marginLeft]).toBe(leftValue);
-        expect(cssPosition.prop("data")[SpacingProperty.marginBottom]).toBe(bottomValue);
-        expect(cssPosition.prop("data")[SpacingProperty.marginRight]).toBe(rightValue);
-        expect(cssPosition.prop("data")[SpacingProperty.paddingTop]).toBe(topValue);
-        expect(cssPosition.prop("data")[SpacingProperty.paddingLeft]).toBe(leftValue);
-        expect(cssPosition.prop("data")[SpacingProperty.paddingBottom]).toBe(bottomValue);
-        expect(cssPosition.prop("data")[SpacingProperty.paddingRight]).toBe(rightValue);
-        expect(typeof cssPosition.prop("onChange")).toEqual(typeof Function);
+        expect((BaseCSSEditor as any).name).toBe(BaseCSSEditor.displayName);
     });
-    test("should execute the `onChange` callback to receive the correct CSSSpacing props", () => {
-        const updatedSpacingValue: string = "5";
-        const topValue: string = "1";
-        const leftValue: string = "2";
-        const callbackSpacing: any = jest.fn(
-            (args: any): void => {
-                expect(args).toEqual({
-                    marginLeft: leftValue,
-                    marginTop: updatedSpacingValue,
-                });
-            }
+    test("should call a registerd callback after a change event", () => {
+        const inputSelector: string = `.${managedClasses.cssEditor_form}`;
+        const onChange: any = jest.fn();
+        const controlled: any = shallow(
+            <BaseCSSEditor managedClasses={managedClasses} onChange={onChange} />
         );
-        const data: CSSEditorValues = {
-            marginTop: topValue,
-            marginLeft: leftValue,
-        };
-        const rendered: any = mount(<CSSEditor data={data} onChange={callbackSpacing} />);
 
-        const renderedCSSSpacing: any = rendered.find("CSSSpacing");
-        const topSpacingInput: any = renderedCSSSpacing.find("input").at(0);
-        topSpacingInput.simulate("change", {
-            target: { value: updatedSpacingValue },
-        });
+        controlled.find(inputSelector).simulate("change");
```

Comment:
This will not scale, since you are adding the color input the color input is being checked here, the test should specify this and look for "CSSColor" and check the onChange against that specific component.